README.md

```diff
@@ -1,2 +1,22 @@
-# year-cohort-projectName
-project template repo
+# 2021-fightingMongooses-gameListing
+
+This project utilises Docker. Visit the [Docker repo](https://hub.docker.com)
```

Comment:
here you should put a link to the iO Academy docker repo instead, since that's the one you're using and our repo has setup instructions

https://github.com/iO-Academy/docker-image

Reply:
Changed!